### ISSUE TITLE
[MISC_5.1.4] Sequi alias quisquam.

### ISSUE THREAD

**Issue by smockle** [smockle/action-filter-issues#2]
Qui quaerat est ea et expedita quia nam sunt explicabo. Nisi beatae dolorem. Sit error in `role="presentation"` autem ut sit enim optio quia. Consequatur perferendis hic quia eius quidem dolorum ad voluptatum aut `<noscript>` within a `<noscript>`. Tenetur consequatur repellendus excepturi quia doloremque. Facere repellat consequuntur harum nesciunt numquam autem nostrum quod.

**Fails**

[4.1.1 Parsing (level A)](https://www.w3.org/WAI/WCAG22/Understanding/parsing)

cc @smockle 

View the [repository](https://github.com/smockle/action-filter-issues).

**Comment by smockle** [comment]
ℹ️ This issue exists for testing the [`smockle/action-filter-issues` action](https://github.com/smockle/action-filter-issues)
